joomla/joomla-cms
# [mod_articles_news] Option Intro/Full Images of the article

Pull Request for Issue #19599 , #16495

### Summary of Changes
Option to use or not, the intro image field or the full image field of the article

### Testing Instructions
1. Go to Extensions ->Modules and create a new Articles - Newsflash
2. Put Show Images option, for any of the 3 options that list. Also, configure the module to display correctly in some position of the template
3. Go to Content -> Articles, and use in one of the articles that the module will show, any of the fields "Intro image" or "Full article image"
4. Go to the frontend, and check depending on the option that you put in the module, it shows or not, the image correctly

### Expected result
The module shows or not, the fields "Intro image" or "Full article image" of the article correctly

### Actual result
The current option only display images in the article context.

### Documentation Changes Required
New module option added.
